src/css/math.less

```diff
@@ -241,14 +241,45 @@
   }
 
   .mq-numerator {
-    padding: 0 0.1em;
+    padding: .1em .1em 0 .1em;
+    width: 100%;
+    margin-bottom: 0.2em;
   }
 
   .mq-denominator {
-    border-top: 1px solid;
-    float: right; // take out of normal flow to manipulate baseline
+    float: left;
     width: 100%;
-    padding: 0.1em;
+    padding: .15em .1em 0 .1em;
+    margin-right: -.1em;
+    margin-left: -.1em;
+    text-align: center;
+  }
+
+  .mq-denominator {
+    border-top: 1px solid;
+  }
+
+  ////
+  // \overset & \underset
+  // overset and underset text
+  .mq-overunder {
+    text-align: center;
+    position: relative;
+    top: .01em;
+
+    // fix \mathbf vertical alignment issue
+    .mq-font {
+      font-family: inherit;
+    }
+
+    &.mq-underset {
+      vertical-align: text-top;
```

Comment:
this doesn't quite work, it will mis-align the underset if the contents are taller than normal text:
![image](https://cloud.githubusercontent.com/assets/225809/13691506/2f0513d8-e6ee-11e5-8bb2-6a8110b3aaa4.png)
For the same LaTeX, `1 + \underset{ abc }{ \frac{2}{3} } + 2`, MathJax renders:
![image](https://cloud.githubusercontent.com/assets/225809/13691627/4cb29616-e6ef-11e5-9b09-aa9ec4f6d36a.png)


I think you'll need the `float` and `width: 100%` trick that fraction uses to do the right thing to the baseline.